### PR TITLE
update instructions for reference software

### LINKED ISSUE #57: Conformance requirements are missing
The spec does not spell out how someone can check if an implementation conforms to the standard or a particular profile

### PR DESCRIPTION
**Pull request by yilun-zhangs**
resolve #57

### CODE REVIEW

File: code/README.md [hunk 1]
```diff
@@ -1,131 +1,221 @@
...
+Building the IAMF library in Microsoft Visual Studio is supported. Visual
+Studio 2015 (14.0) solution has been provided.
+
+Open win32/VS2015/iamf.sln directly.
+
+## Testing the IAMF
+
+### 1. Build application
+
+iamfplayer is is a test application used to decode IAMF bitstream or mp4 file which is IAMF encapsulation.
```
Comment (Frank-Ba) [inline code/README.md:105]: some typos here. How about this:

The iamfplayer is a test application to decode an IAMF bitstream or mp4 file with IAMF encapsulation.

File: code/README.md [hunk 2]
```diff
@@ -1,131 +1,221 @@
...
+    $ cmake -DCMAKE_INSTALL_PREFIX=${BUILD_LIBS} .
+    # ${BUILD_LIBS} is the iamf library and header files installing directory.
+    # If enable binaural rendering, add option `BINAURALIZER`
+
+    $ make
+~~~
+
+### 2. Decoder tests
+
+If want to run binaural cases, please download [default.tf](https://github.com/ebu/bear/releases/download/v0.0.1-pre/default.tf) file into your executing directory.
```
Comment (Frank-Ba) [inline code/README.md:118]: How about replacing the line by:
To produce binaural output, please download the following file and place it in your working directory [default.tf](https://github.com/ebu/bear/releases/download/v0.0.1-pre/default.tf).

File: code/src/iamf_dec/m2m_rdr.c [hunk 3]
```diff
@@ -25,6 +25,11 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+/*
+AOM-IAMF Standard Deliverable Status:
+The following function is out of scope and not part of the IAMF Final Deliverable.
```
Comment (Frank-Ba) [inline code/src/iamf_dec/m2m_rdr.c:30]: Please replace this line by:
This software module is out of scope and not part of the IAMF Final Deliverable.

File: code/README.md [hunk 4]
```diff
@@ -1,131 +1,221 @@
...
+~~~
+
+### Microsoft Visual Studio builds
+
+Building the IAMF library in Microsoft Visual Studio is supported. Visual
+Studio 2015 (14.0) solution has been provided.
+
+Open win32/VS2015/iamf.sln directly.
+
+## Testing the IAMF
```
Comment (Frank-Ba) [inline code/README.md:101]: To address issue #57 , instructions for conformance testing should be included.

**Review by hkchung72 — APPROVED** [review]
it looks good, thanks.

File: code/README.md (hunk 2 above)
Comment (sunghee-hwang) [inline code/README.md:118]:
```suggestion
To produce binaural output, please download the following file and place it in your working directory [default.tf](https://github.com/ebu/bear/releases/download/v0.0.1-pre/default.tf).
```

File: code/src/iamf_dec/m2m_rdr.c (hunk 3 above)
Comment (sunghee-hwang) [inline code/src/iamf_dec/m2m_rdr.c:30]:
```suggestion
This software module is out of scope and not part of the IAMF Final Deliverable.
```

**Review by Frank-Ba — APPROVED** [review]
